app/Console/Kernel.php

```diff
@@ -98,17 +100,34 @@ protected function schedule(Schedule $schedule): void
 
     private function scheduleJobsForCollectionsOrGalleries(Schedule $schedule): void
     {
+
+        if (Config::get('dashbrd.web3_providers.'.FetchCollectionFloorPriceJob::class) === 'opensea') {
+            $schedule
+                ->command(FetchCollectionFloorPrice::class, [
+                    '--limit' => config('services.opensea.rate.max_requests'),
+                ])
+                ->withoutOverlapping()
+                // Opensea allows 4 requests per second, using 5 seconds to leave
+                // some room for other tasks or unexpected delays
+                ->everyFiveSeconds();
```

Comment:
one downside of this is that we will forever queue a job with 5 second intervals to query the database to check if we need to update anything, and it will be no for the majority of the time. For example if we have updated all current collections, then we only want to update this hourly meaning the 5 second interval will run 720 queries before it will retrieve some actual collections to update (hour has passed at that point). At current this won't be that bad as the query is quick, but it's something we can improve upon

Reply:
changing to draft as this bothers me still